client/src/app/appCtrl.js

```diff
@@ -15,8 +15,21 @@ angular.module('app').controller('appCtrl', ['$scope', '$http', '$timeout', 'app
       threshold : appConfig.MAX_FILE_SIZE,
       show : false,
       paused : false,
-      aborted : false
-    }
+      aborted : false,
+    }, 
+    monitor : { // online monitoring
```

Comment:
The $scope.view object is for variables that we need to reference in a template. If the variables are only to keep track of internal state by the module, then we just make them private variables, such as `loadedCSV`, below.